src/app/features/reminders/reminders.component.ts

```diff
@@ -0,0 +1,82 @@
+import { Component, ElementRef, OnInit, ViewChild } from '@angular/core';
+import Status from '../../core/enums/status.enum';
+import { Reminder } from '../../core/models/reminder.model';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './reminders.component.html',
+  styleUrls: ['./reminders.component.scss']
+})
+export class RemindersComponent implements OnInit {
+  reminders: Reminder[];
+  @ViewChild('reminderText') reminderInputRef: ElementRef<HTMLInputElement> = null!;
+
+  sortByDate = (a: Reminder, b: Reminder) => {
```

Comment:
Avoid non used methods in the code.

Reply:
Totally, that was another thing I forgot to remove since I opted to use this arrow function directly where I called it